style.css

```diff
@@ -297,3 +301,122 @@
   color: #7a869a;
   rotate: 90deg;
 }
+
+.contact-h {
+  font-family: " Poppins", sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-p {
+  font-family: " Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #ebebff;
+  width: 100%;
+}
+
+.contact-section {
+  background-color: #6070ff;
+  background-image: url(image-folder/Contact1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position-x: 120%;
+  height: 779px;
+  width: 315px;
+  border-top-left-radius: 100px;
+  padding: 8px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+.contact-section h3 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-left: 40px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  margin-top: 120px;
+}
+
+.form-p {
+  display: flex;
+  margin-left: 12px;
+  margin-right: 20px;
+  margin-bottom: 300px;
+  flex-direction: column;
+}
+
+.input-p {
+  width: 80%;
+  height: 30px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  padding: 15px 38px 15px 16px;
+}
+
+.input-t {
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+  height: 150px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #172b4d;
+  padding: 15px 38px 15px 16px;
+}
```

Comment:
- [ ] [OPTIONAL] Great job 👏 fixing the button styling but please make sure the spacing between the form elements looks similar to the given FIgma design as this ensures your future clients that you can implement a given design down to the last pixel.
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230435040-79106c14-3d82-4b00-bdce-1d22562c0032.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230435346-337801a8-5501-400f-80d9-5a43cd64e5ae.png)